src/main/java/frc/team166/robot/subsystems/Lift.java

```diff
@@ -83,6 +88,8 @@ public double get() {
 
     public Lift() {
```

Comment:
Somewhere in this constructor we need to get the encoder set up:

liftEncoder.setDistancePerPulse(...);
liftEncoder.reset();